ground/src/main/java/com/google/android/ground/ui/map/gms/GoogleMapsFragment.kt

```diff
@@ -255,18 +258,6 @@ class GoogleMapsFragment : SupportMapFragment(), MapFragment {
   override fun moveCamera(point: Point, zoomLevel: Float) =
     getMap().moveCamera(CameraUpdateFactory.newLatLngZoom(point.toLatLng(), zoomLevel))
 
-  private fun addMapPin(mapLocationOfInterest: MapLocationOfInterest, point: Point) {
```

Comment:
We still need to be able to add pins to the map. I think the expectation is that map is smart enough to cluster pins and and polygons at the right zoom levels.